src/main/java/org/carlmontrobotics/commands/AutoMATICALLYGetNote.java

```diff
@@ -53,36 +50,64 @@ public void execute() {
     double angleErrRad = Units.degreesToRadians(LimelightHelpers.getTX(Limelightc.INTAKE_LL_NAME));
     double forwardDistErrMeters = ll.getDistanceToNoteMeters(); 
     double strafeDistErrMeters = forwardDistErrMeters * Math.tan(angleErrRad);
-    // dt.drive(0,0,0);
-    dt.drive(Math.max(forwardDistErrMeters * 2, MIN_MOVEMENT_METERSPSEC),
-        Math.max(strafeDistErrMeters * 2, MIN_MOVEMENT_METERSPSEC), Math.max(angleErrRad * 2, MIN_MOVEMENT_RADSPSEC));
-    // 180deg is about 6.2 rad/sec, min is .5rad/sec
 
     if (LimelightHelpers.getTV(INTAKE_LL_NAME)) {
-      intake.motorSetIntake(0.5);
-    }
-    if(intake.intakeDetectsNote()) {
-      timer.start();
-    } else {
-      timer.stop();
-      timer.reset();
+      dt.drive(forwardDistErrMeters, strafeDistErrMeters, angleErrRad);
     }
-    
+
+    // double forwardSpeed = 0;
+    // double strafeSpeed = 0;
+    // double rotationalSpeed = 0;
+
+    // forwardSpeed = Math.max(
+    // forwardDistErrMeters
+    // * SmartDashboard.getNumber("forward speed multiplier", 1.5),
+    // MIN_MOVEMENT_METERSPSEC);
+
+    // if (LimelightHelpers.getTV(INTAKE_LL_NAME)) {
+    // if (angleErrRad >= 0) {
+    // strafeSpeed = Math.max(
+    // strafeDistErrMeters
+    // * SmartDashboard.getNumber("strafe speed multiplier", 1.5),
+    // MIN_MOVEMENT_METERSPSEC);
+    // rotationalSpeed = Math.max(
+    // angleErrRad
+    // * SmartDashboard.getNumber("rotational speed multiplier", 2),
+    // MIN_MOVEMENT_RADSPSEC);
+    // }
+
+    // else {
+    // strafeSpeed = Math.min(
+    // strafeDistErrMeters
+    // * SmartDashboard.getNumber("strafe speed multiplier", 1.5),
+    // -MIN_MOVEMENT_METERSPSEC);
+    // rotationalSpeed = Math.min(
+    // angleErrRad
+    // * SmartDashboard.getNumber("rotational speed multiplier", 2),
+    // -MIN_MOVEMENT_RADSPSEC);
+    // }
+
+    // if (angleErrRad >= -ERROR_TOLERANCE_RAD
+    // && angleErrRad <= ERROR_TOLERANCE_RAD) {
+    // rotationalSpeed = 0;
+    // }
+
+    // dt.drive(forwardSpeed, strafeSpeed, rotationalSpeed);
+
+
+    // 180deg is about 6.2 rad/sec, min is .5rad/sec
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
     dt.setFieldOriented(true);
-    //SmartDashboard.putBoolean("end", true);
-    intake.stopIntake();
-   
+    // SmartDashboard.putBoolean("end", true);
   }
 
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return (intake.intakeDetectsNote() && timer.get()>0.1);
-    //return timer.get() >= 0.5;
+    return intake.intakeDetectsNote();
```

Comment:
it should be `intake.outtakeDetectsNote();` cuz that's when we know the note is "fully" intaked